### PR TITLE
Add more tasks and redact parser

### PR DESCRIPTION
**Pull request by VyrodovMikhail**

### CODE REVIEW

File: goat/parser/spider_utils.py [hunk 1]
```diff
@@ -190,9 +190,12 @@ def determine_rus_task_type(exam_type: str, topic_id: str) -> tuple[TaskType, bo
     elif exam_type == ExamType.EGE and ((topic_id in ("2", "3", "4", "25")) or (int(topic_id) in range(9, 24))):
         task_type = TaskType.MULT_CHOICE
 
-    elif exam_type == ExamType.OGE and topic_id in ("6", "7", "10", "11"):
+    elif exam_type == ExamType.OGE and topic_id in ("2", "3", "5", "6", "7", "10", "11"):
         task_type = TaskType.MULT_CHOICE
 
+    elif exam_type == ExamType.OGE and topic_id in "4":
+        task_type = TaskType.SOOTV
```
Comment (SpirinEgor) [inline goat/parser/spider_utils.py:197]: `TaskType.MAPPING` or at least correspondence

File: goat/parser/spiders/sdamgia_spider.py [hunk 2]
```diff
@@ -6,6 +6,16 @@
...
+"""            if self.subject == SdamgiaExamSubject.SOC and topic_id in ("24", "25"):
+                not_included = task_container.css("div.probtext")
+                not_included_list = not_included.css("*::text").getall()
+            else:
+                not_included_list = []
+            correct_text_list = []
+            for elem in text_list:
+                if elem not in not_included_list:
+                    correct_text_list.append(elem)"""
+
```
Comment (SpirinEgor) [inline goat/parser/spiders/sdamgia_spider.py:18]: What is that?

**Review by SpirinEgor — CHANGES_REQUESTED** [review]

File: goat/parser/spider_utils.py (hunk 1 above)
Comment (VyrodovMikhail) [reply goat/parser/spider_utils.py]: Done

File: goat/parser/spiders/sdamgia_spider.py (hunk 2 above)
Comment (VyrodovMikhail) [reply goat/parser/spiders/sdamgia_spider.py]: Oops... Deleted

**Review by SpirinEgor — APPROVED** [review]

**Pull request merged**